src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -712,5 +736,14 @@ public override async Task<HealthCheckResult> CheckHealthAsync(CancellationToken
 
             return result ?? await base.CheckHealthAsync(cancellationToken);
         }
+
+        class GeoMasterRetryParam {
```

Comment:
Does these needs to public classes or can they simply be internal or private?

Reply:
I think for nested class default will be private(https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/nested-types)

![image](https://user-images.githubusercontent.com/23129934/113225668-0b00d480-9243-11eb-8799-6ed466944e23.png)

Anyway, I can add private modifier for these class